pkgs/development/python-modules/hdbscan/default.nix

```diff
@@ -44,6 +44,8 @@ buildPythonPackage rec {
     "test_hdbscan_boruvka_balltree_matches"
   ];
 
+  pythonImportsCheck = [ "hdbscan" ];
```

Comment:
Sneaky!

Reply:
Ups, that ended accidentally in that commit.

Reply:
Done.